<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CPU Scheduling</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="readyVLayout">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="readyLabel">
        <property name="text">
         <string>就绪队列</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="readyTableWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>80</number>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>PID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>进程名</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>优先级</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>起址</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>内存大小</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>剩余时间</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>内容</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="CPUSchedulingPushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>CPU调度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="suspendPushButton">
          <property name="text">
           <string>挂起</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0" rowspan="2">
     <layout class="QVBoxLayout" name="backupVLayout">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="backupLabel">
        <property name="text">
         <string>后备队列</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="backupTableWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>80</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>进程名</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>优先级</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>所需内存</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>运行时间</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>内容</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="jobSchedulingPushButton">
        <property name="text">
         <string>作业调度</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2" rowspan="2">
     <layout class="QGridLayout" name="CPUGridLayout">
      <item row="1" column="3">
       <widget class="QTextEdit" name="CPUTextEdit2">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QTextEdit" name="CPUTextEdit1">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="CPULabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>    CPU
----------&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>执行前</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>执行后</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QVBoxLayout" name="suspendVLayout" stretch="0,0,0">
      <property name="spacing">
       <number>7</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="suspendLabel">
        <property name="text">
         <string>挂起队列</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="suspendTableWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>75</number>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>PID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>进程名</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>优先级</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>起址</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>内存大小</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>剩余时间</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>内容</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="recoverPushButton">
        <property name="text">
         <string>解挂</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="2">
     <layout class="QVBoxLayout" name="statusVLayout">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>状态</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="statusTableWidget">
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>75</number>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>PID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>进程名</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>优先级</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>起址</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>内存大小</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>剩余时间</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>内容</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>状态</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QRadioButton" name="autoRadioButton">
        <property name="text">
         <string>自动模式</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="0" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="modelLabel">
        <property name="text">
         <string>模式</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QRadioButton" name="manRadioButton">
        <property name="text">
         <string>手动模式</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="settingsButton">
        <property name="text">
         <string>查看系统设置</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="addJobButton">
        <property name="text">
         <string>添加作业</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="viewRAMButton">
        <property name="text">
         <string>查看内存</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>5</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
